tslint.json

```diff
@@ -108,9 +107,6 @@
     "use-life-cycle-interface": true,
     "use-pipe-transform-interface": true,
     "component-class-suffix": true,
-    "directive-class-suffix": true,
-    "no-access-missing-member": true,
-    "templates-use-public": true,
-    "invoke-injectable": true
```

Comment:
why disable these changes?

Reply:
Because they are not supported by tslint, causing warnings.